Decred Wallet/Features/Backup/BackupReminderViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  BackupReminderViewController.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+import Dcrlibwallet
+import SwiftRichString
+
+class BackupReminderViewController: UIViewController {
+    @IBOutlet weak var label1: UILabel!
+    @IBOutlet weak var label2: UILabel!
+    @IBOutlet weak var label3: UILabel!
+    @IBOutlet weak var label4: UILabel!
+    @IBOutlet weak var label5: UILabel!
+    var checkDict: [Int: Bool] = [:]
```

Comment:
What kind of checks does this dictionary hold. Variable name can be more descriptive.